### PR TITLE
MainSpec is back baby!  Closes #1041

### LINKED ISSUE #1041: Assess/Fix/Implement Ignored Tests in MainSpec
The goal here is to look at the tests in the spec, and determine
1. does the test assess something valuable
2. if no, delete it (with an explanation in the PR)
3. if yes, either fix it up so it passes, or reimplement... whatever is more expedient

### PR DESCRIPTION
**Pull request by mcovarr**

### CODE REVIEW

**Comment by kcibul** [comment]
👍 
looks good to me

**Comment by geoffjentry** [comment]
👍

**Comment by mcovarr** [comment]
oh Travis

File: supportedBackends/local/src/test/scala/cromwell/backend/impl/local/LocalJobExecutionActorSpec.scala [hunk 1]
```diff
@@ -19,11 +19,10 @@ import wdl4s.values._
...
 import scala.concurrent.duration._
 
-class LocalJobExecutionActorSpec extends FlatSpec with BackendTestkitSpec with MockitoSugar with TestFileUtil with TableDrivenPropertyChecks {
+class LocalJobExecutionActorSpec extends FlatSpec with BackendTestkitSpec with MockitoSugar with TestFileUtil
+  with TableDrivenPropertyChecks with HasLocalBackendConfig {
 
-  val globalConfig = ConfigFactory.load()
-  val backendConfig = globalConfig.getConfig("backend.providers.Local.config")
-  val defaultBackendConfigDescriptor = new BackendConfigurationDescriptor(backendConfig, globalConfig)
+  val defaultBackendConfigDescriptor = new BackendConfigurationDescriptor(localBackendConfig, globalConfig = null)
```
Comment (kshakir) [inline supportedBackends/local/src/test/scala/cromwell/backend/impl/local/LocalJobExecutionActorSpec.scala]: Interesting choice on swapping in the `globalConfig = null`. I'm going to pretend this is just like a "mock" and look the other way.

**Comment by kshakir** [comment]
Tried tracing through the latest `CromwellTestkitSpec`, but I'm not entirely sure which of the various `ask` are timing out.

For now 👍  assuming Travis gets happy. I imagine the 10 second default timeouts are just on the cusp of the length of the `SingleToArrayCoercion` wdl runs, and may succeed just in time on a re-run. So maybe kick it a few times until one can go back (again) and figure out what we actually want for the various test durations and blocking within our future/actor/ask/retry soup?

File: supportedBackends/local/src/test/scala/cromwell/backend/impl/local/LocalJobExecutionActorSpec.scala [hunk 2]
```diff
@@ -19,11 +19,10 @@ import wdl4s.values._
 
 import scala.concurrent.duration._
 
-class LocalJobExecutionActorSpec extends FlatSpec with BackendTestkitSpec with MockitoSugar with TestFileUtil with TableDrivenPropertyChecks {
+class LocalJobExecutionActorSpec extends FlatSpec with BackendTestkitSpec with MockitoSugar with TestFileUtil
```
Comment (ruchim) [inline supportedBackends/local/src/test/scala/cromwell/backend/impl/local/LocalJobExecutionActorSpec.scala]: I thought for sure "MockitoSugar" can't be a real thing.....its nice to be proven wrong once in a while 😛

**Comment by mcovarr** [comment]
Replacing with #1057

**Pull request closed without merging**